rpmio/rpmpgp_internal.c

```diff
@@ -1120,6 +1120,12 @@ int pgpPrtParamsSubkeys(const uint8_t *pkts, size_t pktlen,
     return rc;
 }
 
+rpmRC pgpVerifySignature2(pgpDigParams key, pgpDigParams sig, DIGEST_CTX hashctx, char **error_msg)
+{
+    *error_msg = NULL;
```

Comment:
In rpm, these kind of "supplemental" pointers are usually optional, eg

```
    if (error_msg)
        *error_msg = NULL;
```

Reply:
Good catch, I'll fix that!